2021-08-09 15:03:25,380 INFO    MainThread:1030779 [wandb_setup.py:_flush():69] setting env: {'entity': 'katiez', 'project': 'cgan_earth', 'api_key': 'be4ee90fd1224a904f7f4764a16a13960cafe300'}
2021-08-09 15:03:25,380 INFO    MainThread:1030779 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-09 15:03:25,381 INFO    MainThread:1030779 [wandb_init.py:_log_setup():337] Logging user logs to /rds/general/user/jz12018/home/cgan_earth/wandb/run-20210809_150325-1x2460ac/logs/debug.log
2021-08-09 15:03:25,381 INFO    MainThread:1030779 [wandb_init.py:_log_setup():338] Logging internal logs to /rds/general/user/jz12018/home/cgan_earth/wandb/run-20210809_150325-1x2460ac/logs/debug-internal.log
2021-08-09 15:03:25,382 INFO    MainThread:1030779 [wandb_init.py:init():370] calling init triggers
2021-08-09 15:03:25,382 INFO    MainThread:1030779 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-08-09 15:03:25,382 INFO    MainThread:1030779 [wandb_init.py:init():419] starting backend
2021-08-09 15:03:25,383 INFO    MainThread:1030779 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-09 15:03:25,438 INFO    MainThread:1030779 [backend.py:ensure_launched():135] starting backend process...
2021-08-09 15:03:25,485 INFO    MainThread:1030779 [backend.py:ensure_launched():140] started backend process with pid: 1030806
2021-08-09 15:03:25,486 INFO    MainThread:1030779 [wandb_init.py:init():424] backend started and connected
2021-08-09 15:03:25,489 INFO    MainThread:1030779 [wandb_init.py:init():472] updated telemetry
2021-08-09 15:03:25,490 INFO    MainThread:1030779 [wandb_init.py:init():491] communicating current version
2021-08-09 15:03:26,993 INFO    MainThread:1030779 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.11.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-09 15:03:26,994 INFO    MainThread:1030779 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-08-09 15:03:27,205 INFO    MainThread:1030779 [wandb_init.py:init():529] starting run threads in backend
2021-08-09 15:03:27,272 INFO    MainThread:1030779 [wandb_run.py:_console_start():1623] atexit reg
2021-08-09 15:03:27,274 INFO    MainThread:1030779 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2021-08-09 15:03:27,278 INFO    MainThread:1030779 [wandb_run.py:_redirect():1502] Redirecting console.
2021-08-09 15:03:27,281 INFO    MainThread:1030779 [wandb_run.py:_redirect():1558] Redirects installed.
2021-08-09 15:03:27,282 INFO    MainThread:1030779 [wandb_init.py:init():554] run started, returning control to user process
2021-08-09 15:03:27,284 INFO    MainThread:1030779 [wandb_config.py:__setitem__():141] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x148d6163d438>>
2021-08-09 15:03:27,285 INFO    MainThread:1030779 [wandb_run.py:_config_callback():872] config_cb no_cuda False None
2021-08-09 15:03:27,287 INFO    MainThread:1030779 [wandb_config.py:__setitem__():141] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x148d6163d438>>
2021-08-09 15:03:27,287 INFO    MainThread:1030779 [wandb_run.py:_config_callback():872] config_cb seed 42 None
2021-08-09 15:03:27,288 INFO    MainThread:1030779 [wandb_config.py:__setitem__():141] config set log_interval = 100 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x148d6163d438>>
2021-08-09 15:03:27,288 INFO    MainThread:1030779 [wandb_run.py:_config_callback():872] config_cb log_interval 100 None
2021-08-09 15:03:27,289 INFO    MainThread:1030779 [wandb_watch.py:watch():40] Watching
2021-08-09 15:03:27,291 INFO    MainThread:1030779 [wandb_watch.py:watch():40] Watching
2021-08-09 15:03:27,432 INFO    MainThread:1030779 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-08-09 15:03:27,436 INFO    MainThread:1030779 [wandb_run.py:_restore():1565] restore
2021-08-09 15:03:29,797 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2021-08-09 15:03:30,099 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 742
}

2021-08-09 15:03:30,238 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 3394
}

2021-08-09 15:03:30,342 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 742
  total_bytes: 3394
}

2021-08-09 15:03:30,451 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2735
  total_bytes: 3394
}

2021-08-09 15:03:30,561 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:30,665 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:30,769 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:30,873 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:31,226 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:31,330 INFO    MainThread:1030779 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3394
  total_bytes: 3394
}

2021-08-09 15:03:32,483 INFO    MainThread:1030779 [wandb_run.py:_show_files():1937] logging synced files
